templates/example_spatial_analysis_script.ipynb

```diff
@@ -149,13 +149,13 @@
     "\n",
     "# For cluster spatial analysis, first find all the cell phenotypes in the data to use as an input for the clustergram\n",
     "pheno_titles = all_data[\"cell_type\"].drop_duplicates()\n",
-    "visualize.visualize_z_scores(stats_cluster.loc['fov6', \"z\", :, :].values, pheno_titles)\n",
+    "visualize.visualize_z_scores(stats_cluster.loc['fov6', \"z\", :, :].values, pheno_titles, pheno_titles)\n",
     "# For channel spatial analysis, first find all the marker titles to use as input for the clustergram\n",
     "channel_start = np.where(all_data.columns == settings.PRE_CHANNEL_COL)[0][0] + 1\n",
     "channel_end = np.where(all_data.columns == settings.POST_CHANNEL_COL)[0][0]\n",
     "\n",
     "marker_titles = all_data.iloc[:, channel_start:channel_end].drop(excluded_channels, axis=1).columns\n",
-    "visualize.visualize_z_scores(stats_channel.loc['fov6', \"z\", :, :].values, marker_titles)"
+    "visualize.visualize_z_scores(stats_channel.loc['fov6', \"z\", :, :].values, marker_titles, marker_titles)"
```

Comment:
Two things:

- Split up this cell into cluster spatial analysis and channel spatial analysis visualizations
- Can you display the resulting visualization for both cells? Currently, it only saves them.